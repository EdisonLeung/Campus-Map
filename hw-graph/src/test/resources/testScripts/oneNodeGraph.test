# Tests adding one node to Graph

# Create Graph
CreateGraph oneNode

# Add one node
AddNode oneNode a

# Print the nodes and children of that node
ListNodes oneNode
ListChildren oneNode a