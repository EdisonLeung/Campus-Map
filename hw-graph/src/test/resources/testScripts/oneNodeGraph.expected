# Tests adding one node to Graph

# Create Graph
created graph oneNode

# Add one node
added node a to oneNode

# Print the nodes and children of that node
oneNode contains: a
the children of a in oneNode are: